dashboard/src/utils/mockData.js

```diff
@@ -0,0 +1,32 @@
+export const MOCK_DATA = [
```

Comment:
I'm still a bit concerned about the generic name `MOCK_DATA` ... this mocks the response of `/datasets/list`, but we'll certainly want to have other mocks in the future to test additional parts of the dashboard.

Reply:
The `/datasets/list` response format has changed. There's now a "results" field in the JSON as well as a "next_url" for paging. Also, the dashboard now relies on the `resource_id`.

```
{
  "next_url": "",
  "results": [
    {
      "metadata": {
        "dataset.created": "2022-07-27T15:39:06+00:00"
      },
      "name": "pbench-user-benchmark_xyz.22_2022.07.27T15.39.06",
      "resource_id": "c54ab5929dc6230937402b682751c1ac"
    },
    {
      "metadata": {
        "dataset.created": "2022-07-27T15:01:57+00:00"
      },
      "name": "Short timer",
      "resource_id": "8c19617fc80580eaf6bb6808a1f8c46f"
    },
    {
      "metadata": {
        "dataset.created": "2022-07-27T13:45:02+00:00"
      },
      "name": "Soon FOUR",
      "resource_id": "4e28f76ccc9572f7e6a27898678db39d"
    },
    {
      "metadata": {
        "dataset.created": "2022-07-27T13:45:11+00:00"
      },
      "name": "Soonish",
      "resource_id": "892e506d3302ebaa466b9bc4c02f1780"
    },
    {
      "metadata": {
        "dataset.created": "2022-07-20T16:28:32+00:00"
      },
      "name": "Soon ONE",
      "resource_id": "6a4b55f64f8dd5f997be0da70a5e0d2c"
    },
    {
      "metadata": {
        "dataset.created": "2022-07-27T13:44:53+00:00"
      },
      "name": "Soon THREE",
      "resource_id": "e368132b16f8a45407c77c0a54a427fb"
    }
  ],
  "total": 6
}
```